docs/proposals/user-namespace.md

```diff
@@ -0,0 +1,226 @@
+<!-- BEGIN MUNGE: UNVERSIONED_WARNING -->
```

Comment:
please remove the  UNVERSIONED_WARNING block in its entirety.  it is obsolete now.